### PR TITLE
Refactor app settings

### PR DESCRIPTION
**Pull request by elgatovital**
Changes proposed in this pull request:
- Revert xmas images & launcher icons
- Make app build on apple silicon
- Refactor app settings

before   | after  
---|---
image | image

@gnosis/mobile-devs

### CODE REVIEW

File: app/src/main/java/io/gnosis/safe/di/components/ViewComponent.kt [hunk 1]
```diff
@@ -123,6 +120,8 @@ interface ViewComponent {
 
     fun inject(fragment: GetInTouchFragment)
 
+    fun inject(fragment: AboutSafeFragment)
```
Comment (biafra23) [inline app/src/main/java/io/gnosis/safe/di/components/ViewComponent.kt:123]: Where is this requirement coming from? Is there a ticket for this?

File: app/src/main/java/io/gnosis/safe/ui/settings/app/AppSettingsFragment.kt [hunk 2]
```diff
@@ -16,6 +16,7 @@ import io.gnosis.safe.di.components.ViewComponent
 import io.gnosis.safe.ui.base.fragment.BaseViewBindingFragment
 import io.gnosis.safe.ui.settings.SettingsFragmentDirections
 import pm.gnosis.svalinn.common.utils.openUrl
```
Comment (biafra23) [inline app/src/main/java/io/gnosis/safe/ui/settings/app/AppSettingsFragment.kt:18]: Seems like several imports can be removed. For example this one.

File: buildsystem/versions.gradle [hunk 3]
```diff
@@ -16,7 +16,7 @@ ext {
             androidx_test_ext               : '1.1.3',
             androidx_recycler_view          : '1.1.0',
             androidx_swipe_refresh          : '1.1.0-rc01',
-            androidx_room                   : '2.2.5',
+            androidx_room                   : '2.4.0-alpha03',
```
Comment (biafra23) [inline buildsystem/versions.gradle:19]: 👍🏻

**Review by biafra23 — APPROVED** [review]
LGTM. (Except for the unnecessary imports as mentioned)

File: app/src/main/java/io/gnosis/safe/di/components/ViewComponent.kt (hunk 1 above)
Comment (elgatovital) [reply app/src/main/java/io/gnosis/safe/di/components/ViewComponent.kt:123]: same settings structure as in ios app

Comment (elgatovital) [reply app/src/main/java/io/gnosis/safe/di/components/ViewComponent.kt:123]: image

**Pull request merged**